Sources/TuistKit/Utils/XCResultParser.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import Mockable
+import Path
+import XCResultKit
+
+@Mockable
+public protocol XCResultParsing {
+    func parse(path: AbsolutePath) -> ParsedXCResult?
+}
+
+public struct ParsedXCResult {
```

Comment:
```suggestion
public struct XCResult {
```

nit: I'd get rid of the `Parsed` prefix